src/protocol/payload/mod.rs

```diff
@@ -241,3 +241,10 @@ pub fn read_timestamp<B: Buf>(bytes: &mut B) -> io::Result<OffsetDateTime> {
     OffsetDateTime::from_unix_timestamp(timestamp_i64)
         .map_err(|_| io::Error::new(io::ErrorKind::InvalidData, "Bad UTC timestamp"))
 }
+
+/// Reads a timestamp from the bytes.
```

Comment:
Suggestion:
```suggestion
/// Reads a timestamp encoded as 4 bytes.
```

Ditto for the 8-byte variant.